### PR TITLE
summary and indicator, canvas fields.

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/36991

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/IncidentFields/incidentfield-EmailCampaignCanvas.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "markdown",
+    "unmapped": false,
+    "unsearchable": false,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0"
```
Comment (roysagi) [inline Packs/CommonTypes/IncidentFields/incidentfield-EmailCampaignCanvas.json:29]: Let's make sure with @idovandijk on which version these additions are made.

**Review by roysagi — APPROVED** [review]
Looks good - please see my comment below

File: Packs/CommonTypes/IncidentFields/incidentfield-EmailCampaignCanvas.json (hunk 1 above)
Comment (idovandijk) [reply Packs/CommonTypes/IncidentFields/incidentfield-EmailCampaignCanvas.json:29]: 6.0.0

**Review by idovandijk — CHANGES_REQUESTED** [review]
Please let me see the final result before merging

**Review by idovandijk — APPROVED** [review]
Itay changed the way summary is displayed, looks great. Also tested on the server and works fine

**Pull request merged**